contracts/script/deploy/DeployPR1T1XChainReader.s.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.25;
+
+import { ProxyAdmin } from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import { TransparentUpgradeableProxy } from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+import { DeploymentUtils } from "../lib/DeploymentUtils.sol";
+
+import { T1XChainReader } from "../../src/libraries/xChain/T1XChainReader.sol";
+import { T1Constants } from "../../src/libraries/constants/T1Constants.sol";
+
+contract DeployPR1T1XChainReader is DeploymentUtils {
+    uint32 internal constant ORIGIN_CHAIN = uint32(T1Constants.T1_DEVNET_CHAIN_ID);
```

Comment:
If this is meant to be deployed on a partner rollup then using `T1_DEVNET_CHAIN_ID` is not correct.

Also I think local domain would be a more apt name.

Reply:
Upon further inspection, we are not using localDomain anywhere. I think it could be removed.